packages/probing-frontend/src/utils/date.utils.ts

```diff
@@ -0,0 +1,11 @@
+export function formatDateString(dateString: string) {
+  try {
+    const formatter = new Intl.DateTimeFormat('it', {
+      dateStyle: 'short',
+      timeStyle: 'short',
+    })
+    return formatter.format(new Date(dateString))
+  } catch {
+    return 'N/D'
+  }
```

Comment:
I would remove the try/catch here

Reply:
I implemented the try-catch block because the function accepts `dateString: string` as a parameter, and if the `dateString` received isn't a valid date, it will cause the app to crash.
 
 We could check if dateString it's a valid date to avoid the try catch?
 
```
 const date = new Date(dateString);
 const isValidDate = !isNaN(date.getTime());
```

Reply:
The crash here would help us recognize quickly that something is wrong with what the backend is sending to us, so it's fine.